src/components/ArchiveBody/ArchiveBody.js

```diff
@@ -0,0 +1,33 @@
+import React, {useState} from 'react';
+import teams from '../../data/teams'
+import TeamButton from '../TeamButton/TeamButton'
+import ArchiveCommittee from '../CommitteeArchive/ArchiveCommittee'
+import "./ArchiveBody.css";
+
+function ArchiveBody() {
+
+  const [committee, setCommittee] = useState("studio")
+
+  const refArray = ["studio", "icpc", "design", "cyber", "teachla", "w", "ai", "hack"]
```

Comment:
do you mind changing these values to match the `source` names in the resources (as well as the names in `teams.js`)? currently none of the resources show up in the ArchiveCommittee because of the committee name mismatch. so for example, change "hack" to "Hack" and "teachla" to "Teach LA"